app/Controllers/SettingsController.php

```
@@ -208,6 +222,90 @@ public function updateEmailSettings(Request $request, Response $response, mysqli
+                $insert = $db->prepare(
+                    'INSERT INTO registrazione_campi (etichetta, tipo, obbligatorio, attivo, ordine)
+                     SELECT ?, ?, ?, 1, COALESCE(MAX(ordine), 0) + 1 FROM registrazione_campi'
+                );
+                if ($insert !== false) {
+                    $insert->bind_param('ssi', $newLabel, $newType, $newRequired);
+                    $insert->execute();
+                    $insert->close();
+                }
+            }
+        } catch (\Throwable $e) {
+            \App\Support\SecureLogger::error('[Settings] custom registration fields save failed', ['error' => $e->getMessage()]);
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\App\Support\SecureLogger' in method 'saveRegistrationCustomFields'. (undefined)

(StaticAccess)